apps/builder/app/builder/shared/html-editor.tsx

```diff
@@ -70,68 +70,47 @@ const wrapperStyle = css({
   position: "relative",
   // 1 line is 16px
   // set min 10 lines and max 20 lines
-  "--ws-code-editor-min-height": "160px",
-  "--ws-code-editor-max-height": "320px",
+  ...getMinMaxHeightVars({ minHeight: "160px", maxHeight: "320px" }),
```

Comment:
implementation details got hidden and there is no variable names duplication/copy paste